packaging/installer/methods/packages.md

```diff
@@ -76,18 +118,46 @@ Available groups:
 | `repoconfig` | Configuration packages        |
 | `devel`      | Dev builds (ignore)           |
 
-Supported distributions:
+### Supported Distributions
+
+Within each repository group, you'll find directories for specific distributions:
+
+- `debian`: For Debian Linux and compatible distributions
+- `ubuntu`: For Ubuntu Linux and compatible distributions
```

Comment:
```suggestion
- `debian`: For Debian Linux and binary-compatible distributions
- `ubuntu`: For Ubuntu Linux and binary-compatible distributions
```

Binary compatibility as compared to just ‘compatibility’ is actually really significant here, because there are a number of Debian and Ubuntu derived distributions that are notably _not_ binary compatible with Debian or Ubuntu respectively, and Netdata will thus not work on them.